apps/payment/models.py

```diff
@@ -298,9 +297,32 @@ def save(self, *args, **kwargs):
                 raise NotAcceptable("Insufficient funds")
 
             self.user.save()
-        super(PaymentTransaction, self).save(*args, **kwargs)
+        super().save(*args, **kwargs)
 
     class Meta:
         ordering = ['-datetime']
         verbose_name = _('transaksjon')
         verbose_name_plural = _('transaksjoner')
+
+
+class PaymentReceipt(models.Model):
+    """Transaction receipt"""
+    receipt_id = models.CharField(max_length=128, default=None)
```

Comment:
Maybe use UUIDField for this? Generates unique IDs for us. Can also be used as primary key (`primary_key=True`).